include/z64.h

```diff
@@ -402,7 +402,7 @@ typedef struct PlayState {
     /* 0x11D44 */ s32 (*isPlayerDroppingFish)(struct PlayState* play);
     /* 0x11D48 */ s32 (*startPlayerFishing)(struct PlayState* play);
     /* 0x11D4C */ s32 (*grabPlayer)(struct PlayState* play, Player* player);
-    /* 0x11D50 */ s32 (*startPlayerCutscene)(struct PlayState* play, Actor* actor, s32 csAction);
+    /* 0x11D50 */ s32 (*tryPlayerCsAction)(struct PlayState* play, Actor* actor, s32 csAction);
```

Comment:
my first thought when seeing this name was that
```suggestion
    /* 0x11D50 */ s32 (*playerTryCsAction)(struct PlayState* play, Actor* actor, s32 csAction);
```
would be more in line with `Player_TryCsAction`
but I guess this follows the existing names like `play->startPlayerFishing = Player_StartFishing;`

Reply:
Id be fine with eventually changing all of them to match that scheme

Really these are all player callbacks. It might not be "real", but what if we grouped all of these as a substruct and let that name hold the "this is a player callback" information?

Reply:
Im imagining like `play->playerCallback.grab();`, `play->playerCallback.tryCsAction();`, `play->playerCallback.startFishing();`

Reply:
seems good, let's see that another time